tests/layer_tests/tensorflow_tests/test_tf_SegmentSum.py

```diff
@@ -69,3 +69,55 @@ def test_segment_sum_different_types(self, params, ie_device, precision, ir_vers
         self._test(*self.create_segment_sum_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_legacy_frontend=use_legacy_frontend)
+
+
+class TestSegmentSumComplex(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        rng = np.random.default_rng()
+        assert "data_real:0" in inputs_info
+        assert "data_imag:0" in inputs_info
+        assert "segment_ids:0" in inputs_info
+        data_real_shape = inputs_info["data_real:0"]
+        data_imag_shape = inputs_info["data_imag:0"]
+        segment_ids_shape = inputs_info["segment_ids:0"]
+        inputs_data = {}
+        inputs_data["data_real:0"] = rng.random(data_real_shape).astype(np.float32)
+        inputs_data["data_imag:0"] = rng.random(data_imag_shape).astype(np.float32)
+        inputs_data["segment_ids:0"] = rng.integers(
+            low=0, high=data_real_shape[0], size=segment_ids_shape
+        ).astype(np.int32)
```

Comment:
segment_ids data must be sorted according to TensorFlow SegmentSum specification

Reply:
Applied `np.sort`